test/unit/specs/components/scroll-button.spec.js

```diff
@@ -27,9 +27,8 @@ describe('Scroll button', () => {
   });
 
   it('should scroll the window up when clicked', () => {
-    const mockMethods = {
-      scrollToTop: jest.fn(),
-    };
+    const mockMethods = jest.fn();
```

Comment:
I'm not sure I understand why you changed mockMethods to a single mock function? Also, why is creating a new global variable with the same value as the mockMethods variable necessary? Why not just reference the mockMethods variable?

Also, the name "mockMethods" doesn't really make sense now that its value is a single mocked method instead of a set of mocked methods